CEAP.md

```diff
@@ -29,10 +29,12 @@ Artigo 2 - Atende as seguintes despesas:
 * Contratação **para apoio da atividade parlamentar** de consultoria e trabalhos técnicos, permitindo pesquisa socioeconômica
 * Divulgação da atividade parlamentar, **exceto nos 120 dias anteriores à eleição federal, estadual ou municipal**, exceto se o deputado não for candidato.
 * Participação em cursos, palestras, seminários, simpósios, congressos ou eventos do mesmo gênero, **realizado por instituição especializada, até o limite de 25% do valor da menor cota mensal (R$ 7697.16) por mês**
-* Complementação do auxílio moradia (hotel ou aluguel), **até R$ 1747,00 reais mensais**]
```

Comment:
Would you mind sending this minor edits (we love them!) in separate PR so it's easily to merge without back and forth code reviewing? Just for brevity sake ; )

(I mean, all minor edits in `CEAP.md`, not only this line).